testfiles/RadLoHydrHeatCoolAuto.idf

```diff
@@ -2,7 +2,7 @@
 ! Basic file description:  1 story building divided into 3 interior conditioned zones. Roof with no plenum.
 !                          No ground contact with floor.
 !
-! Highlights:              Basic test of the hydronic radiant heating and cooling system with autosizing
+! Highlights:              Basic test of the hydronic radiant heating and cooling system with scalable and autosizing
```

Comment:
I suggest changing these lines to read "Basic test of hydronic radiant heating and cooling systems with area-scaled sizing and autosizing"